sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -857,6 +857,47 @@ class HiveQuerySuite extends HiveComparisonTest with SQLTestUtils with BeforeAnd
     assert(sql(s"list file $testFile").count() == 1)
   }
 
+  test("ADD ARCHIVE command") {
+    withTempDir { dir =>
+      val file1 = File.createTempFile("someprefix1", "somesuffix1", dir)
+      val zipFile = new File(dir, "test.zip")
+      val file2 = File.createTempFile("someprefix2", "somesuffix2", dir)
+      // Emulate unsupported archive format with .bz2 suffix.
+      val unsupportedArchive = new File(dir, "test.bz2")
```

Comment:
Could you split this single test unit into smaller meaningful units? e.g.,
```
test("ADD ARCHIVE/LIST ARCHIVES commands") { ... }
test("ADD ARCHIVE command - unsupported archive formats") { ... }
...
```

Reply:
Thanks for the advice. I've split it.